src/component/tabs.js

```diff
@@ -100,17 +101,30 @@ export const ComponentTabs = ({
     })
   }, [featureRegistrationContext])
 
+  React.useEffect(() => {
+    return registerCallbackHandler({
+      featureRegistrationContext: featureRegistrationContext,
+      featureName: features.FINDING_CONFIGURATIONS,
+      callback: ({feature}) => setFindingCfgsFeature(feature),
+    })
+  }, [featureRegistrationContext])
+
   const handleChange = (_, newView) => {
     searchParamContext.update({'view': newView})
   }
 
+  // only show compliance tab if there are finding cfgs available
```

Comment:
might be a bit implicit to omit the tab, also I'd like to avoid movement in UI elements; how about disabling instead and adding a meaningful hover text?